modules/Microsoft.VirtualMachineImages/imageTemplates/.test/common/dependencies.bicep

```diff
@@ -0,0 +1,69 @@
+@description('Optional. The location to deploy resources to.')
+param location string = resourceGroup().location
+
+@description('Required. The name of the Shared Image Gallery to create.')
+param galleryName string
+
+@description('Required. The name of the Image Definition to create in the Shared Image Gallery.')
+param sigImageDefinitionName string
+
+@description('Required. The name of the Managed Identity to create.')
+param managedIdentityName string
+
+resource managedIdentity 'Microsoft.ManagedIdentity/userAssignedIdentities@2018-11-30' = {
+    name: managedIdentityName
+    location: location
+}
+
+resource gallery 'Microsoft.Compute/galleries@2022-03-03' = {
+    name: galleryName
+    location: location
+    properties: {
+    }
+  }
+
+  module roleAssignment 'dependencies_rbac.bicep' = {
+    name: '${deployment().name}-MSI-roleAssignment'
+    scope: subscription()
+    params: {
+      managedIdentityPrincipalId: managedIdentity.properties.principalId
+      managedIdentityResourceId: managedIdentity.id
+    }
+  }
```

Comment:
This looks slightly indented on the right, I'd suggest formatting the entire file

Reply:
solved.